modules/vslam/orb_slam.cpp

```diff
@@ -0,0 +1,552 @@
+#include <opencv2/opencv.hpp>
+#include <opencv2/objdetect.hpp>
+#include <opencv2/imgcodecs.hpp>
+#include <opencv2/highgui/highgui.hpp>
+#include <opencv2/imgproc/imgproc.hpp>
+#include <iostream>
+#include <opencv2/videoio.hpp>
+#include <opencv2/core.hpp>
+#include <stdlib.h>
+#include <stdio.h>
+#include <ctime>
+#include <opencv2/core/utility.hpp>
+#include <opencv2/sfm.hpp>
+#include <opencv2/viz.hpp>
+
+using namespace cv;
+using namespace std;
```

Comment:
It's not recommended to use `using namespace std`. With `using namespace cv` - OK